redhat-developer/app-services-cli
# feat: generate credentials for Java Kafka client

This PR adds an additional flag to `rhoas service-account create` and `rhoas service-account reset-credentials` for supporting `properties` file compatible with Java Kafka client.

### Verification Steps

1. Run the following command:
```
./rhoas service-account create --file-format java-kafka-properties --short-description java-properties
```
2. It should create a file `credentials.properties` with credentials and callback URL.

### Type of change
